src/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSource.cs

```diff
@@ -12,13 +12,19 @@ public partial class DiagnosticListener : System.Diagnostics.DiagnosticSource, S
     public static IObservable<DiagnosticListener> AllListeners { get { throw null; } } 
     public virtual void Dispose() { }
     public override bool IsEnabled(string name) { throw null; }
+    public override bool IsEnabled(string name, object arg1) { throw null; }
```

Comment:
Drop this overload.  Instead add a default parameter to the 3 argument version.   It has the same effect.